### PR TITLE
Add unit test for the `server/sandbox_remove`.

### PR DESCRIPTION
**Pull request by utam0k**
Signed-off-by: utam0k <k0ma@utam0k.jp>

#### What type of PR is this?

/kind other

#### What this PR does / why we need it:

Add unit test for the `server/sandbox_remove` to increase code coverage.

#### Which issue(s) this PR fixes:

None

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by haircommander** [comment]
/ok-to-test
/approve

thanks, LGTM

**Comment by utam0k** [comment]
@haircommander Can I fix the `shfmt` error?

**Comment by haircommander** [comment]
no need it'll be fixed by https://github.com/cri-o/cri-o/pull/5316

**Review by fgiudici — APPROVED** [review]
LGTM (just found a typo in a message)

File: server/sandbox_remove_test.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	// Prepare the sut
+	BeforeEach(func() {
+		beforeEach()
+		setupSUT()
+	})
+
+	AfterEach(afterEach)
+
+	t.Describe("PodSandboxRemove", func() {
+		It("should failed when sandbox is not created", func() {
```
[the remark this answers is not in the document]
Comment (utam0k) [reply server/sandbox_remove_test.go]: @fgiudici Thanks. I have fixed it.

**Comment by utam0k** [comment]
> Hey @utam0k , the last commit looks a bit weird to me (the merge one).
> My guess is that you intended to rebase your branch on the main branch... if so, you may want probably to:
> `git rebase origin/main`
> and than force push your branch again.

Ops... I have fixed it.

**Comment by haircommander** [comment]
One more nit, could you squash the two commits into one @utam0k ?

**Comment by utam0k** [comment]
> One more nit, could you squash the two commits into one @utam0k ?

Sure!

**Pull request merged**